src/Data/Env.elm

```diff
@@ -21,6 +22,11 @@ betagouvUrl =
     "https://beta.gouv.fr/startups/ecobalyse.html"
 
 
+cguUrl : String
+cguUrl =
+    "https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire-draft/old/pages-textiles-old/conditions-dutilisation"
```

Comment:
It's a bit strange to link to page url containing the word "old". We should probably ask this page to move to something more permanent.